src/api/queries/__generated__/channels.generated.tsx

```diff
@@ -105,6 +107,11 @@ export const BasicChannelFieldsFragmentDoc = gql`
     handle
     avatarPhotoUrl
     createdAt
+    description
```

Comment:
This breaks the consumer views in mocked version since we're not returning this info